AEPMessaging/Sources/schemas/InAppSchemaData.swift

```diff
@@ -76,6 +87,24 @@ public class InAppSchemaData: NSObject, Codable {
     }
 }
 
+extension InAppSchemaData {
```

Comment:
nit: We can eliminate duplicate methods and just expose `asArray` and `asDictionary` here. And use `if/guard let` at the time of invocation.
For example:
```
var asArray: [Any]? {
    content as? [Any]
}
```

During invocation
```
if let arrValue = asArray {
    try container.encode(AnyCodable(arrValue), forKey: .content)
} 
```